schema.sql

```diff
@@ -1,13 +1,21 @@
-/* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
   id INTEGER PRIMARY KEY,
   name VARCHAR(255),
   date_of_birth DATE,
   escape_attempts INTEGER,
   neutered BOOLEAN,
-  weight_kg DECIMAL(10,2)
+  weight_kg DECIMAL(10,2),
+  species_id INTEGER REFERENCES species(id),
+  owner_id INTEGER REFERENCES owners(id)
 );
 
-ALTER TABLE animals
-ADD COLUMN species VARCHAR(255);
\ No newline at end of file
+CREATE TABLE species (
+  id INTEGER PRIMARY KEY,
+  name VARCHAR(255)
+);
+CREATE TABLE owners (
+  id INTEGER PRIMARY KEY,
+  full_name VARCHAR(255),
+  age INTEGER
+);
```

Comment:
- [ ] Good job adding the owners and species table, how ever here we should also add the statements that modify the animals table by dropping the species column and adding the species and owners id as foreign keys.

e.g.
```
   ALTER TABLE animals DROP column species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```